orchestra/todos/views.py

```diff
@@ -91,3 +74,17 @@ def perform_update(self, serializer):
             'complete' if todo.completed else 'incomplete')
         message_experts_slack_group(
             todo.task.project.slack_group_id, message)
+
+
+class TodoListTemplateDetail(generics.RetrieveUpdateAPIView):
+    permission_classes = (permissions.IsAuthenticated,)
+
+    serializer_class = TodoListTemplateSerializer
+    queryset = TodoListTemplate.objects.all()
+
+
+class TodoListTemplateList(generics.ListCreateAPIView):
```

Comment:
we probably just want list, but not create?

Reply:
@marcua Create will be required to allow designers to save todolist templates from the user interface.